manifests/base/jupyter-minimal-notebook-imagestream.yaml

```diff
@@ -16,24 +16,37 @@ spec:
   # N Version of the image
   - annotations:
       opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
-      opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version": "3.5"}, {"name": "Notebook","version": "6.5"}]'
+      opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version": "3.6"}]'
```

Comment:
Is 3.6 correct? It doesn't correspond with the annotation below (and in other imagestreams)

Reply:
Hey @jstourac :)
I have got this version from [here](https://github.com/opendatahub-io/notebooks/blob/2023b/jupyter/minimal/ubi9-python-3.9/Pipfile#L11).